manual/manual/tutorials/coreexamples.etex

```diff
@@ -409,11 +416,12 @@ name_of_binary_digit 0;;
 name_of_binary_digit (-1);;
 \end{caml_example}
 
-The "with" part is actually a regular pattern-matching on the
-exception value. Thus, several exceptions can be caught by one
+The "with" part does pattern matching on the
+exception value with the same syntax and behavior as "match". Thus,
+several exceptions can be caught by one
 "try"\ldots"with" construct. Also, finalization can be performed by
 trapping all exceptions, performing the finalization, then raising
-again the exception:
+the exception again:
```

Comment:
Given that we now have functions with the name, how about "then re-raising the exception:"? (I refer to `Printexc.raise_with_backtrace`)